components/normalizer/src/properties.rs

```diff
@@ -89,6 +90,49 @@ impl CanonicalComposition {
     ///
     /// [📚 Help choosing a constructor](icu_provider::constructors)
     #[cfg(feature = "compiled_data")]
+    pub const fn new() -> Self {
```

Comment:
For existing borrowed types we use `Owned::new() -> Borrowed`. This keeps the constructors together. I think we should discuss if that's what we want to do, but for now do it for consistency.

Reply:
I'm warming up to the idea that it should be `Borrowed::new()` because we want to be able to cfg-out the whole owned class at some point.

Reply:
https://github.com/unicode-org/icu4x/issues/5440